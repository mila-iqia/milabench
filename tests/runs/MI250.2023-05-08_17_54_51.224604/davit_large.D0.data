{"event": "config", "data": {"dirs": {"base": "/milabench/envs", "venv": "/milabench/envs/venv/torch", "data": "/milabench/envs/data", "runs": "/milabench/envs/runs", "extra": "/milabench/envs/extra/timm", "cache": "/milabench/envs/cache"}, "arch": "rocm", "group": "timm", "install_group": "torch", "install_variant": "rocm", "run_name": "redizilo.2023-05-08_17:54:51.224604", "enabled": true, "capabilities": {"nodes": 1}, "max_duration": 600, "voir": {"options": {"stop": 60, "interval": "1s"}}, "config_base": "/milabench/milabench/config", "config_file": "/milabench/milabench/config/standard.yaml", "definition": "/milabench/milabench/benchmarks/timm", "plan": {"method": "per_gpu"}, "argv": {"--amp": true, "--model": "davit_large", "--batch-size": 128, "--lr-base": 0.01}, "tags": ["classification", "transformer", "vision"], "weight": 1.0, "name": "davit_large", "tag": ["davit_large", "D0"], "device": 0, "devices": [0], "env": {"ROCR_VISIBLE_DEVICES": "0"}}, "pipe": null}
{"event": "start", "data": {"command": ["voir", "--config", "/milabench/envs/extra/timm/voirconf-davit_large.D0-0efae956f1553a76c1e03985181900f5.json", "/milabench/milabench/benchmarks/timm/pytorch-image-models/train.py", "--amp", "--model", "davit_large", "--batch-size", "128", "--lr-base", "0.01", "--data-dir", "/milabench/envs/data", "--dataset", "FakeImageNet", "--output", "/milabench/envs/extra/timm/redizilo.2023-05-08_17:54:51.224604/davit_large.D0", "--checkpoint-hist", "1"], "time": 1683570775.2934234}, "pipe": null}
{"event": "phase", "data": {"name": "init"}, "pipe": "data"}
{"event": "phase", "data": {"name": "parse_args"}, "pipe": "data"}
{"event": "phase", "data": {"name": "load_script"}, "pipe": "data"}
{"event": "phase", "data": {"name": "run_script"}, "pipe": "data"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "Training with a single process on 1 device (cuda:0).\n", "pipe": "stderr"}
{"event": "line", "data": "Model davit_large created, param count:196811752\n", "pipe": "stderr"}
{"event": "line", "data": "Data processing configuration for current model + dataset:\n", "pipe": "stderr"}
{"event": "line", "data": "\tinput_size: (3, 224, 224)\n", "pipe": "stderr"}
{"event": "line", "data": "\tinterpolation: bicubic\n", "pipe": "stderr"}
{"event": "line", "data": "\tmean: (0.485, 0.456, 0.406)\n", "pipe": "stderr"}
{"event": "line", "data": "\tstd: (0.229, 0.224, 0.225)\n", "pipe": "stderr"}
{"event": "line", "data": "\tcrop_pct: 0.95\n", "pipe": "stderr"}
{"event": "line", "data": "\tcrop_mode: center\n", "pipe": "stderr"}
{"event": "line", "data": "Learning rate (0.005) calculated from base learning rate (0.01) and global batch size (128) with linear scaling.\n", "pipe": "stderr"}
{"event": "line", "data": "Using native Torch AMP. Training in mixed precision.\n", "pipe": "stderr"}
{"event": "line", "data": "Scheduled epochs: 300. LR stepped per epoch.\n", "pipe": "stderr"}
{"event": "data", "data": {"task": "main", "gpudata": {"0": {"memory": [839, 65520], "load": 0.0, "temperature": 33.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.224273681640625}, "pipe": "data"}
{"event": "line", "data": "Train: 0 [   0/32 (  0%)]  Loss: 7.224 (7.22)  Time: 5.476s,   23.38/s  (5.476s,   23.38/s)  LR: 1.000e-05  Data: 0.743 (0.743)\n", "pipe": "stderr"}
{"event": "data", "data": {"task": "main", "gpudata": {"0": {"memory": [14380, 65520], "load": 0.41, "temperature": 35.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.1763715744018555}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"0": {"memory": [32306, 65520], "load": 0.46, "temperature": 38.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.276237964630127}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.335600852966309}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"0": {"memory": [32306, 65520], "load": 1.0, "temperature": 39.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.255938529968262}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"0": {"memory": [32306, 65520], "load": 1.0, "temperature": 40.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.162021636962891}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"0": {"memory": [32306, 65520], "load": 1.0, "temperature": 41.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.256057262420654}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.43942648582422, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [1, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"0": {"memory": [32306, 65520], "load": 0.48, "temperature": 41.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.163331031799316}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.383124020976105, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [2, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.308176040649414}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"0": {"memory": [32308, 65520], "load": 1.0, "temperature": 41.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.44700402775919, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [3, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.283428192138672}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"0": {"memory": [32308, 65520], "load": 1.0, "temperature": 43.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.261251285742645, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [4, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.234744071960449}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"0": {"memory": [32308, 65520], "load": 1.0, "temperature": 42.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.37007803399633, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [5, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.196105480194092}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.37782957661751, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [6, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"0": {"memory": [32308, 65520], "load": 0.46, "temperature": 42.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.325323104858398}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.39364417444012, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [7, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.243505477905273}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"0": {"memory": [32308, 65520], "load": 1.0, "temperature": 43.0}}}, "pipe": "data"}
{"event": "line", "data": "Memory access fault by GPU node-2 (Agent handle: 0x7eba240) on address 0x7fb95b200000. Reason: Unknown.\n", "pipe": "stderr"}
{"event": "end", "data": {"command": ["voir", "--config", "/milabench/envs/extra/timm/voirconf-davit_large.D0-0efae956f1553a76c1e03985181900f5.json", "/milabench/milabench/benchmarks/timm/pytorch-image-models/train.py", "--amp", "--model", "davit_large", "--batch-size", "128", "--lr-base", "0.01", "--data-dir", "/milabench/envs/data", "--dataset", "FakeImageNet", "--output", "/milabench/envs/extra/timm/redizilo.2023-05-08_17:54:51.224604/davit_large.D0", "--checkpoint-hist", "1"], "time": 1683570828.350233, "return_code": -6}, "pipe": null}
