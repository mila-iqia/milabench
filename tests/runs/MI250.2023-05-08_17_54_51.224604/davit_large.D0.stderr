/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
Training with a single process on 1 device (cuda:0).
Model davit_large created, param count:196811752
Data processing configuration for current model + dataset:
	input_size: (3, 224, 224)
	interpolation: bicubic
	mean: (0.485, 0.456, 0.406)
	std: (0.229, 0.224, 0.225)
	crop_pct: 0.95
	crop_mode: center
Learning rate (0.005) calculated from base learning rate (0.01) and global batch size (128) with linear scaling.
Using native Torch AMP. Training in mixed precision.
Scheduled epochs: 300. LR stepped per epoch.
Train: 0 [   0/32 (  0%)]  Loss: 7.224 (7.22)  Time: 5.476s,   23.38/s  (5.476s,   23.38/s)  LR: 1.000e-05  Data: 0.743 (0.743)
Memory access fault by GPU node-2 (Agent handle: 0x7eba240) on address 0x7fb95b200000. Reason: Unknown.
